src/it/hello-world-web-components/src/main/java/example/helloworld/HelloWorld.java

```diff
@@ -0,0 +1,29 @@
+package example.helloworld;
+
+import elemental2.dom.HTMLElement;
+import elemental2.dom.HTMLTemplateElement;
+import jsinterop.annotations.JsType;
+
+import static elemental2.dom.DomGlobal.customElements;
+import static elemental2.dom.DomGlobal.document;
+
+@JsType
+public class HelloWorld extends HTMLElement {
```

Comment:
probably worth fleshing out a bit more as a test, use some of the lifecycle methods and ensure they aren't pruned or renamed in such a way that they will not be called (i suspect they will without a goog.exportSymbol call, but I could be mistaken).